src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -500,7 +500,14 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
     userEvent.click(await screen.findByText(/Choose a provider/i));
 
     // Choose Provider based on current location
-    userEvent.click(await screen.findByText(/use your current location$/i));
+    const currentLocButton = await screen.findByText(
+      /use your current location$/i,
+    );
+    await screen.findByText(/Displaying 1 to /i);
```

Comment:
Figuring this out resulted in the most console.log statements I've ever added to the app at once.

When running this page in JSDOM, it would sometimes catch a weird in between state where we were showing the link to switch your sort as well as the loading indicator for providers. Normally when you're using the page you don't see that, either because of the state update batching or because it's too quick. So this test would switch to the current location during that weird in between state and get out of sync.

I suspect we could fix this in the component itself, but I don't think it can actually impact users.